### DISCUSSION TITLE
Is this project dead?

### DISCUSSION THREAD

**Discussion by douglasware** [microsoft/BotFramework-Composer#9661] [General]
Considering the dependency on .NET Core 3.1 which has not been fixed in well over a year and the lack of new commits, it sure seems like maybe it is dead.

**Comment by Berandomsen** [comment]
Is it dead ? The question is , does copilot studio support multilingual bots in its new iteration , and the answer appears to be no. “PVA” bots are monolingual, and manual in effort. For every advance , converting new innovations into a multilingual experience is an afterthought. 

Please someone correct me .

**Reply by RupenAnjaria** [reply]
I feel the same, it would be better if they close the project so that we don't run here and there for issues.

**Comment by kcabral817** [comment]
It's not dead. It's just a smoldering dumpster fire.

**Comment by ZepHakase22** [comment]
I agree that Composer is dead! 
But also Bot Framework must not feel too good!!!
